dts/arm/ti/am64x_m4.dtsi

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2023 Texas Instruments Incorporated
```

Comment:
This is an almost identical copy of the am62x_m4. Should I keep this Copyright or change it?

Reply:
@dnltz There is no need to retain the copyright in this particular case. However we need some strategy to not duplicate these SoC level dtsi files (atleast for different cores in same SoC), in this case it is okay.